src/napari/_qt/widgets/qt_welcome.py

```diff
@@ -190,6 +208,10 @@ def dropEvent(self, event):
         self._update_property('drag', False)
         self.sig_dropped.emit(event)
 
+    def _next_tip(self, event=None):
```

Comment:
```suggestion
    def showEvent(self, event):
        super().showEvent(event)
        self._tip_timer.start()

    def hideEvent(self, event):
        super().hideEvent()
        self._tip_timer.stop()

    def _next_tip(self, event=None):
```

Use timer only when needed. 
We need to think how to prevent of starting timer when testing. To prevent segfaults.